apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableVersionRange.cs

```diff
@@ -33,6 +33,41 @@ public BigtableVersionRange(BigtableVersion? start, BigtableVersion? end)
         }
 
         /// <summary>
+        /// Creates a new <see cref="BigtableVersionRange"/>.
+        /// </summary>
+        /// <param name="start">
+        /// Inclusive lower bound non-negative version value, or -1 to initialize from the
+        /// microseconds of DateTime.UtcNow. If null, interpreted as 0.
+        /// </param>
+        /// <param name="end">
+        /// Exclusive upper bound non-negative version value, or -1 to initialize from the
+        /// microseconds of DateTime.UtcNow. If null, interpreted as infinity.
+        /// </param>
+        public BigtableVersionRange(long? start, long? end)
+        {
+            throw new NotImplementedException();
+        }
+
+        // TODO: Get clarification on use cases to see how useful this would be.
+        /// <summary>
+        /// Creates a new <see cref="BigtableVersionRange"/>.
+        /// </summary>
+        /// <param name="startTimestamp">
+        /// Inclusive lower bound timestamp whose microseconds should be used as the version value.
+        /// If null, interpreted as 0. It is recommended that timestamps be specified in UTC so there
```

Comment:
I'd *require* the Kind to be UTC to be honest. Better to alert callers to problems early, IMO.

Reply:
Done